doc/en/user/source/data/webadmin/layers.rst

```diff
@@ -297,6 +297,7 @@ Sets the WFS specific publishing parameters.
 
 * **Per-Request Feature Limit**—Sets the maximum number of features for a layer a WFS GetFeature operation should generate (regardless of the actual number of query hits)
 * **Maximum number of decimals**—Sets the maximum number of decimals in GML output.
+* **Activate complex to simple features conversion** - Enables the conversion of complex features to simple features, using only SF-0 (simple) attributes for compatible output formats like CSV, KML, SHAPE-ZIP.
```

Comment:
One might wonder why this is even needed. I'd at least suggest to explain that it would come with some loss of information. Or it could be made even simpler, and just happen if the target output format is not able to handle complex features natively, the current alternative is a hard exception... what would be the harm?